.github/workflows/deploy.yml

```diff
@@ -0,0 +1,84 @@
+        name: Wait for other inprogress deployment runs
+        env:
+          GITHUB_TOKEN: ${{ secrets.API_TOKEN }}
+
+      - name: Start ${{ github.event.inputs.environment }} Deployment
+        uses: bobheadxi/deployments@v0.4.2
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.API_TOKEN }}
+          env: ${{ github.event.inputs.environment }}
+          ref: ${{ github.event.inputs.sha }}
+
+
+
+      - name: print env variables
+        run: |
+          GIT_SHA_SHORT=$(echo ${GITHUB_SHA} | cut -c 1-7)
+          echo "SHA_TAG=$GIT_SHA_SHORT" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v2
+      - name: Pin Terraform version
+        uses: hashicorp/setup-terraform@v1.2.1
+        with:
+          terraform_version: 0.13.5
+
+      - name: Install Terraform CloudFoundry Provider
+        run: |
+            mkdir -p $HOME/.terraform.d/plugins/linux_amd64
+            cd $HOME/.terraform.d/plugins/linux_amd64
+            wget -q -O zipfile https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/releases/download/v0.12.6/terraform-provider-cloudfoundry_0.12.6_linux_amd64.zip && unzip zipfile && rm zipfile
+
+      - name: Terraform init
+        run: |
+          echo $TF_VAR_paas_app_docker_image
+          terraform init -backend-config=terraform/workspace-variables/qa_backend.tfvars terraform
+        env:
+          TF_VAR_paas_app_docker_image: ${{env.DOCKER_REPO}}:${{ env.SHA_TAG }}
+          ARM_ACCESS_KEY: ${{ secrets.ARM_ACCESS_KEY }}
+
+      #- name: Terraform Plan and Apply
+      #  run: |
+      #    terraform plan -var-file=terraform/workspace-variables/qa.tfvars terraform
+      #    terraform apply -var-file=terraform/workspace-variables/qa.tfvars -auto-approve terraform
+      #  env:
+      #    TF_VAR_paas_app_docker_image: ${{env.DOCKER_REPO}}:${{ env.SHA_TAG }}
+      #    ARM_ACCESS_KEY: ${{ secrets.ARM_ACCESS_KEY }}
+      #    TF_VAR_paas_password: ${{ secrets.TF_VAR_PAAS_PASSWORD }}
+      #    TF_VAR_paas_user: ${{ secrets.TF_VAR_PAAS_USER }}
+      #    TF_VAR_paas_log_url: ${{ secrets.TF_VAR_PAAS_LOGGING_ENDPOINT_PORT }}
+      #    TF_VAR_paas_settings__basic_auth__password: ${{ secrets.TF_VAR_PAAS_SETTINGS__BASIC_AUTH__PASSWORD }}
+      #    TF_VAR_paas_settings__dttp__client_secret:  ${{ secrets.TF_VAR_PAAS_SETTINGS__DTTP__CLIENT_SECRET }}
+      #    TF_VAR_paas_settings__basic_auth__username :  ${{ secrets.TF_VAR_PAAS_SETTINGS__BASIC_AUTH__USERNAME }}
+      #    TF_VAR_paas_settings__dttp__client_id :  ${{ secrets.TF_VAR_PAAS_SETTINGS__DTTP__CLIENT_ID }}
+      #    TF_VAR_paas_settings__dttp__tenant_id :  ${{ secrets.TF_VAR_PAAS_SETTINGS__DTTP__TENANT_ID }}
+      #    TF_VAR_paas_settings__dttp__api_base_url :  ${{ secrets.TF_VAR_PAAS_SETTINGS__DTTP__API_BASE_URL }}
+      #    TF_VAR_dockerhub_username: ${{ secrets.DOCKERHUB_USERNAME }}
+      #    TF_VAR_dockerhub_password: ${{ secrets.DOCKERHUB_PERSONAL_ACCESS_TOKEN }}
```

Comment:
uncomment

Reply:
and instead of `${{ env.SHA_TAG }}` just use `${{ github.event.inputs.sha }}`

Reply:
done..